README.md

```diff
@@ -51,28 +43,34 @@ if (scope != null) {
 
 ### Starting a new Span
 
-The common case starts an `Scope` that's automatically registered for intra-process propagation via `ScopeManager`. The best practice is to use a try-with-resources pattern which handles Exceptions and early returns:
+The common case starts an `Scope` that's automatically registered for intra-process propagation via `ScopeManager`.
+Note that the default behaviour of `startActive()` does not finish the span on `Scope.close()`.
+This was made on purpose because `try-with-resources` construct would finished the span before
```

Comment:
"This decision was made on purpose because the `try-with-resources` construct would finish the span ..."